### ISSUE TITLE
HTML language tags missing

### ISSUE THREAD

**Issue by juyrjola** [City-of-Helsinki/servicemap#292]
Sivun kieltä ei ole määritetty HTML-koodissa. Tästä voi aiheutua tilanne, jossa ruudunlukuohjelman puhesyntetisaattori valitsee käyttöönsä sovelluksen oletuskielen, joka on usein englanti. Tämän jälkeen ruudunlukija lukee sivujen sisällön väärällä kielellä, jota on mahdoton ymmärtää. Peruskäyttäjä ei myöskään usein osaa vaihtaa syntetisaattorin kieltä, vaan palvelun käyttö muuttuu tällöin mahdottomaksi.

WCAG: 3.1.1 Sivun kieli: Jokaisen verkkosivun oletusarvoinen luonnollinen kieli voidaan
selvittää ohjelmallisesti.

Korjausehdotus: lisätään HTML-koodiin header-osioon määritys ”lang=fi”.

**Comment by villeristi** [comment]
Language-tag gets implemented to DOM on app.start -stage, alltough it's not visible when the page source is being viewed.

![lang-tag](https://cloud.githubusercontent.com/assets/2413192/10135308/66130eb8-65f5-11e5-8431-a41660eb9a28.png)
